include/swift/Remote/MetadataReader.h

```diff
@@ -756,6 +757,9 @@ class MetadataReader {
       return BuiltOpaque;
     }
     }
+
+    std::cerr << "fatal error: " << "Unhandled MetadataKind in switch." << "\n";
+    std::abort();
```

Comment:
Ah right.  I think that having a `swift_unreachable` is going to be cleaner.  We could model it after `llvm_unreachable`.